### DISCUSSION TITLE
支持Windows SMTC协议

### DISCUSSION THREAD

**Discussion by e8xl** [SPlayer-Dev/SPlayer#191] [想法 Ideas]
如题
[与系统媒体传输控件集成](https://learn.microsoft.com/zh-cn/windows/uwp/audio-video-camera/integrate-with-systemmediatransportcontrols)
SMTC还支持输出高清的图片信息 如果能支持就更好了
方便更多软件读取播放歌曲信息 类似Wallpaper Engine等

**Comment by L7132** [comment]
原来这个叫SMTC啊😍

**Comment by imsyy** [comment]
会在重构后支持

**Reply by ChengzZg** [reply]
smtc能打开，但是输出高清封面选项是黑的，而且不能点是什么原因呢，win11
